core/domain/blog_validators.py

```diff
@@ -117,4 +199,11 @@ def _get_external_id_relationships(cls, item):
                 [item.id]
             )
         ]
+        field_name_to_external_model_references.append(
+            base_model_validators.UserSettingsModelFetcherDetails(
+                'editor_ids', item.editor_ids,
+                may_contain_system_ids=True,
+                may_contain_pseudonymous_ids=False
+            )
+        )
```

Comment:
Can you check whether we need any custom validator for this model?

Reply:
I don't think any custom validation is required as we are already validating the id, and the user model foe editor ids.